optimum/exporters/onnx/config.py

```diff
@@ -65,6 +65,7 @@ class TextDecoderOnnxConfig(OnnxConfigWithPast):
 
     PAD_ATTENTION_MASK_TO_PAST = True
     DUMMY_INPUT_GENERATOR_CLASSES = (DummyTextInputGenerator, DummyPastKeyValuesGenerator)
+    DUMMY_PKV_GENERATOR_CLASS = DummyPastKeyValuesGenerator
```

Comment:
Why?

Reply:
Because Bloom uses its own custom past key values generator, and accessing through `self.DUMMY_INPUT_GENERATOR_CLASSES[1]` in the parent class grabs the wrong input generator for bloom. WDYT? Is it ok?

Reply:
Yes, so I see it is in `TextDecoderOnnxConfig`, which is nice. The whole idea with this 3 level hierarchy is to also keep flexibilty. This makes sense here IMO.